Makefile

```diff
@@ -44,12 +45,31 @@ $(call add-bindata,assets,./bindata/...,bindata,assets,pkg/operator/assets/binda
 $(call build-image,cert-manager-operator,$(IMAGE_OPERATOR),./images/ci/Dockerfile,.)
 $(call build-image,cert-manager-operator-bundle,$(IMAGE_OPERATOR_BUNDLE),./bundle/bundle.Dockerfile,./bundle)
 
+# generate scorecard image targets
+.PHONY: build/custom-scorecard-tests
+build/custom-scorecard-tests:
+	go build -mod=vendor $(GO_GCFLAGS) $(GO_ASMFLAGS) -o $(BUILD_DIR)/$(@F) ./test
+
+.PHONY: build-e2e-image
+build-e2e-image:
+	docker build -f ./images/custom-scorecard-tests/Dockerfile ./ -t quay.io/ytripath/cert-manager-operator-e2e-tests:latest
```

Comment:
We should parametrize image coordinates

Reply:
Do you mean changing `latest` to `$(IMAGE_TAG)`

Reply:
Changed image coordinates to variables

Reply:
Per convention:
```
Usage:  docker build [OPTIONS] PATH | URL | -
```
It is better to put your `./` at the end.